packages/circuit-ui/components/Selector/Selector.tsx

```diff
@@ -137,8 +128,28 @@ const noMarginStyles = ({ noMargin }: LabelElProps) => {
   `;
 };
 
+const sizeStyles = ({ theme, size = 'mega' }: LabelElProps & StyleProps) => {
+  const sizeMap = {
+    kilo: {
+      padding: `${theme.spacings.bit} ${theme.spacings.mega}`,
+    },
+    mega: {
+      padding: `${theme.spacings.byte} ${theme.spacings.giga}`,
+    },
+    flexible: {
+      padding: `${theme.spacings.mega} ${theme.spacings.mega}`,
+    },
+  };
+
+  return css({
+    label: `selector--${size}`,
```

Comment:
I think this would override the label in baseStyles, I would just declare it there 🙂 (baseStyles are receiving the size as well, if we want to add it to the label)

Reply:
True, will do 👍

Reply:
Realized that it's actually should override the parent label as the one for size is more specific and includes the `selector` part.

Reply:
Sounds good, then maybe we should make it `selector__label--${size}` for consistency